src/components/ImageArray.tsx

```diff
@@ -95,7 +95,14 @@ export const ImageArray: React.FC<Props> = ({
           まだ画像がありません
         </Typography>
       ) : (
-        <ImageList variant="quilted" cols={4} rowHeight={200} sx={{ m: 0 }}>
+        <ImageList
+          variant="quilted"
+          sx={{
+            m: 1,
+            gridTemplateColumns:
+              "repeat(auto-fill, minmax(150px, 1fr)) !important",
+          }}
+        >
```

Comment:
以下のように書くのではうまくいかなさそうですか？

```suggestion
        <ImageList
          variant="standard"
          cols={4}
        >
```